### DISCUSSION TITLE
Human skin color applies to "Human_damage" and "Human_damageoverlay" sprites

### DISCUSSION THREAD

**Discussion by Dr-Psycho-SK** [FakeFishGames/Barotrauma#13648] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I was working in Sub editor when I took notice of the fact that the "Human_damage" and "Human_damageoverlay" was being affected by the selected skin color of the character.

Needless to say why but I think all the human characters are supposed to have the same blood color and so I assume it's a bug/oversight.

### Reproduction steps

For best results:
1. Find a white skinned character.
2. Find one or more darker skinned characters.
3. Injure them both enough to see the damage overlay clearly and compare the visuals.

![image_2024-03-21_211943856](https://github.com/FakeFishGames/Barotrauma/assets/135719646/26ff8438-e3aa-41d8-909a-04c29daba517)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
None.
```

**Comment by Dr-Psycho-SK** [comment]
When I stated that Human Skin color applies to the overlay images, it was just my best deduction, in truth if that's not the reason for the blood color change then I don't know.

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, coming up in the next unstable build!
